cmd/admin/cert_test.go

```diff
@@ -253,13 +257,21 @@ func TestRevokeSerials(t *testing.T) {
 	// Revoking a doomed-to-fail serial should also result in one log line.
 	mra.reset()
 	log.Clear()
-	mra.doomedToFail = []string{"bar"}
+	mra.doomedToFail = []string{"048c3f6388afb7695dd4d6bbe3d264f1e5e5"}
 	err = a.revokeSerials(context.Background(), serials, 0, false, false, 1)
 	test.AssertNotError(t, err, "")
 	test.AssertEquals(t, len(log.GetAllMatching("failed to revoke")), 1)
 	test.AssertEquals(t, len(mra.revocationRequests), 3)
 	assertRequestsContain(mra.revocationRequests, 0, false, false)
 
+	// Revoking a correctly-formatted serial with extraneous characters should get carried through
+	mra.reset()
+	log.Clear()
+	err = a.revokeSerials(context.Background(), serials, 1, true, true, 3)
+	test.AssertNotError(t, err, "")
+	test.AssertEquals(t, len(mra.revocationRequests), 3)
+	assertRequestsContain(mra.revocationRequests, 1, true, true)
+
```

Comment:
This standalone test case is not necessary, it is identical to the test case below it. Because you modified the list of serials at the top of this function to have extraneous characters in them, _every_ test case within this function is already testing your new functionality.